README.id.md

```diff
@@ -0,0 +1,203 @@
+
+_Baca ini dalam bahasa lain: [English](README.en.md) [На русском](README.md)_
+
+## Kontribusi
+
+&#x1F49A; Mulai [disini](https://github.com/naXa777/spring-cloud-example/issues).
+
+## Deskripsi
+
+> [Cloud-native](https://pivotal.io/de/cloud-native) merupakan pendekatan yang digunakan untuk membangun dan menjalankan aplikasi yang memanfaatkan keuntungan dari model komputasi awan.
+
+> [Microservices](https://pivotal.io/microservices) merupakan pendekatan yang digunakan untuk mengembangkan sebuah aplikasi sebagai suatu koleksi dari servis-servis kecil; setiap servis ini mengimplementasikan kapabilitas bisnis, jalan dalam proses tersendiri dan berkomunikasi melalui API HTTP. Setiap microservice dapat dideploy, upgrade, scale, dan restart namun tidak bergantung pada servis lain pada aplikasi tersebut, yang biasanya merupakan bagian dari sistem otomata.
+
+[(c) Pivotal](https://pivotal.io/de/cloud-native)
+
+Kita membagi aplikasi besar menjadi bagian-bagian yang lebih kecil yang disebut 'services'. Dan setiap dari servis ini dapat dideploy atau scale secara tersendiri._Bagaimana suatu servis mengetahui yang lainnya?_
+
+### Penemuan servis
+
+|      Modul       |           URL             |
+| :--------------: | :-----------------------: |
+| discovery-server | `http://host:8761/eureka` |
+
+Terdapat beberapa cara Anda dapat menemukan servis di Spring Cloud:
+
+* Spring Cloud [Consul](https://cloud.spring.io/spring-cloud-consul/)
+* Spring Cloud [Zookeeper](https://cloud.spring.io/spring-cloud-zookeeper/)
+* Spring Cloud [Netflix](https://cloud.spring.io/spring-cloud-netflix/)
+
+Projek ini secara spesifik fokus ke poin terakhir, yaitu projek Netflix Spring Cloud. Media service streaming berbasis awan yang dimiliki oleh Amerika Serikat, yang dinamakan Netflix ini, telah membuka konten framework dan alat-alat microservicenya.
+
+Demi kesederhanaan, terdapat satu instansi server Eureka. Maka server Eureka tidak dikonfigurasi untuk mendaftarkan dirinya sendiri dengan peer yang lain.
+Anda mungkin ingin memiliki banyak server discovery demi ketersediaan saat produksi, maka ubah properti berikut:
+
+    eureka.client.register-with-eureka=true
+    eureka.client.fetch-registry=true
+
+#### Ketersediaan Tinggi 
+
+Eureka memastikan secara periodik bahwa servis aplikasi yang di return atau dikembalikan ke client dalam kondisi baik dan available. Dan ini juga memastikan bahwa dalam waktu tertentu ketika Server Discovery nya down, semua client masih dapat berjalan. 
+
+Eureka dibangun dengan pemikiran ketersediaan yang tinggi:
+
+ * Registrynya terdistribusi (local cache pada setiap client).
+ * Client  _dapat_ beroperasi tanpa server discovery (apabila servernya down).
+ * Client mengambil deltas untuk memperbarui registry.
+
+#### Dashboard
+
+Dashbord Eureka berbasis web terenabled sebagai default. Ini menunjukkan informasi-informasi penting seperti metadata servis.
```

Comment:
please double check this line

Reply:
Thanks for the heads up, it's the translation that's as accurate as possible. For example, the word `default` does not have a direct translation to Indonesia. Closest thing possible is probably `biasanya/awalnya` which if translated back, is `usually/in the beginning`. I've never seen and heard someone prefer `dasbor` over `dashboard`, so it's not changing, too.